resources/keywords/provision_dbinstance.resource

```diff
@@ -8,12 +8,20 @@ Library     ../../utils/scripts/util.py
 *** Variables ***
 ${engine}     ${EMPTY}
 ${invalid}      False
+${pstgInstClass}    db.r5.xlarge
+${allocStorage}    20
 
 *** Keywords ***
 User Navigates To Create Database Instance Screen From Database Access Page
     User Navigates To Database Access Under Data Services
     Wait Until Page Contains Element    ${titleDatabaseAccess}
-    Select Create Database Instance From Configure Button DropDown
+    Select Create Database Instance From Configuration Button DropDown Provision
+
+Select ${option} From ${menu} Button DropDown Provision
```

Comment:
Duplicate

Reply:
This is the same as my previous comment. The first one was removed because of the current bug found. It will be returned when the bug will be fixed.

Reply:
Please review with earlier comments